applications/StructuralMechanicsApplication/custom_conditions/surface_load_condition_3d.h

```diff
@@ -154,16 +154,16 @@ class KRATOS_API(STRUCTURAL_MECHANICS_APPLICATION)  SurfaceLoadCondition3D
 
     void CalculateAndSubKp(
         Matrix& K,
-        array_1d<double, 3>& ge,
-        array_1d<double, 3>& gn,
+        const array_1d<double, 3>& ge,
+        const array_1d<double, 3>& gn,
         const Matrix& DN_De,
         const Vector& N,
-        double pressure,
-        double weight );
+        const double Pressure,
+        const double Weight );
 
     void MakeCrossMatrix(
```

Comment:
now the same function another time? (it may well be my fault...still it is not nice)